pkgs/by-name/mm/mmseqs2/package.nix

```diff
@@ -58,17 +57,19 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.cmakeFeature "CMAKE_CUDA_ARCHITECTURES" cudaPackages.flags.cmakeCudaArchitecturesString)
   ];
 
-  buildInputs =
-    lib.optionals stdenv.cc.isClang [
-      llvmPackages.openmp
-      zlib
-      bzip2
-    ]
-    ++ lib.optional enableMpi mpi
-    ++ lib.optionals cudaSupport [
-      cudaPackages.cuda_cudart
-      cudaPackages.cuda_cccl
-    ];
+  buildInputs = [
+    zstd
+  ]
+  ++ lib.optionals stdenv.cc.isClang [
+    llvmPackages.openmp
+    zlib
+    bzip2
```

Comment:
(FWIW, I doubt these are only required on Clang – more likely it falls back to vendored versions and it just so happens that Clang is exposing build failures in older vendored versions of these?

Not a blocker, of course, since it’s pre‐existing.)